src/main/java/lawrence/task/TaskList.java

```diff
@@ -0,0 +1,167 @@
+package lawrence.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the list of tasks the user has specified to be tracked by the chatbot.
+ */
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
Maybe specify what kind of list? It should also be plural since it is a collection.
e.g. tasklists or tasks

Reply:
Yup, you're right. Will amend this accordingly!